src/ngraph/ngraph_compiler.cc

```diff
@@ -434,8 +434,17 @@ void Compiler::CheckInNgraph() {
             }
           }
         }
-      } else if (ngraph_log_verbose()) {
-        unsupported_op_names.insert(node->operation_);
+      } else {
+        if (ngraph_log_verbose()) {
+          unsupported_op_names.insert(node->operation_);
+        }
+
+        if (ngraph_log_verbose_detail()) {
```

Comment:
shouldnt `unsupported_op_names.insert` be in this too

Reply:
That occurred to me, but I don't think so:

- It's not really clear whether or not `ngraph_log_verbose()` and `ngraph_log_verbose_detail()` are intended to indicate basically two different levels of logging.  (Their names imply yes, but the fact that they're two independent flags suggests no.)

- It seems useful to be able to control these reports independently.

But I'm not passionate about the issue; happy to change this if you still want me to.

Reply:
i am referring to `unsupported_op_names`, why do we not use that `set` to eliminate duplicate prints for this new verbose detail log

Reply:
I see.  Yeah, I was thinking it's a hassle to do that, but I think you're right.  Will change.

Reply:
Actually, I just remembered on of my reasons: Suppose we have 2 operator types that are unsupported, but we have a large network with hundreds of occurrences of those two operators.

We could end up with hundreds of distinct entries in that set, even though only two operators are mentioned.  So it will be more difficult to do the kind of analysis that I've done over the past few weeks.

Reply:
I see. we could add hash(op.name+attrs) to the unordered_set to eliminate duplicates (we already have hashing on op attr in util) . I am ok as-is if this is too much work for log, but deep.learning models tend to repeat the layers (with same attrs) quite often , even in small networks.